incubator/jupyter-notebook/jupyter-notebook/templates/service.yaml

```
@@ -0,0 +1,16 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "jupyter.fullname" . }}
+spec:
+  type: NodePort
```

```suggestion
  type: ClusterIP
```
Change ServiceType to ClusterIP